worker/worker.go

```diff
@@ -62,6 +91,36 @@ func HandleJoinResponse(response *utils.HttpResponse, w *Worker) {
 	w.QueueId = queueId
 }
 
+func (w *Worker) GetTask(serverEndPoint string) (*Task, error) {
```

Comment:
hum ... in the future, we might want to not mix the communication details (the rest stuff) with the, say, business logic. I think we can postpone this but, please, annotate this suggestion somewhere.

Reply:
I'll keep it in my notes, we can discuss this further.